fixes.inc

```diff
@@ -9398,6 +9515,78 @@ native BAD_DestroyActor(actorid) = DestroyActor;
 #endif
 
 /*
+ * FIXES_SetVehicleParamsEx(vehicleid, engine, lights, alarm, doors, bonnet, boot, objective)
+ *
+ * FIXES:
+ *     OnPlayerEnterVehicle_3
+ */
+
+#if defined _ALS_SetVehicleParamsEx
+	#error _ALS_SetVehicleParamsEx defined
+#endif
+native BAD_SetVehicleParamsEx(vehicleid, engine, lights, alarm, doors, bonnet, boot, objective) = SetVehicleParamsEx;
+
+#if FIX_OnPlayerEnterVehicle_3
+	stock FIXES_SetVehicleParamsEx(vehicleid, engine, lights, alarm, doors, bonnet, boot, objective)
+	{
+		FIXES_gsVehicleIsLocked[vehicleid] = doors == VEHICLE_PARAMS_ON;
+		return SetVehicleParamsEx(vehicleid, engine, lights, alarm, doors, bonnet, boot, objective);
+	}
+
+	#define _ALS_SetVehicleParamsEx
+	#define SetVehicleParamsEx( FIXES_SetVehicleParamsEx(
+#endif
+
+/*
+ * FIXES_GetVehicleParamsEx(vehicleid, &engine, &lights, &alarm, &doors, &bonnet, &boot, &objective)
+ *
+ * FIXES:
+ *     OnPlayerEnterVehicle_3
+ */
+
+#if defined _ALS_GetVehicleParamsEx
+	#error _ALS_GetVehicleParamsEx defined
+#endif
+native BAD_GetVehicleParamsEx(vehicleid, &engine, &lights, &alarm, &doors, &bonnet, &boot, &objective) = GetVehicleParamsEx;
+
+#if FIX_OnPlayerEnterVehicle_3
+	stock FIXES_GetVehicleParamsEx(vehicleid, engine, lights, alarm, doors, bonnet, boot, objective)
+	{
+		return GetVehicleParamsEx(vehicleid, engine, lights, alarm, doors, bonnet, boot, objective);
```

Comment:
Yes totally, thank you that didn't work.
https://github.com/Open-GTO/sa-mp-fixes/pull/64/commits/8e91421fb316098e8ca262412ec54b4cb9c49d0c